Gr4vy UIKit Sample App/CheckoutViewController.swift

```diff
@@ -89,6 +89,9 @@ class CheckoutViewController: UIViewController {
                 case .cancelled:
                     print("User cancelled")
                     return
+                case .cardDetailsChanged(let bin, let cardType, let scheme):
+                    print("Card details changed, BIN: \(bin), Card Type: \(cardType), Scheme: \(scheme ?? "Unknown")")
+                    return
```

Comment:
Example usage. The internal example app will print this while debugging in Xcode

![Screenshot 2025-07-10 at 11 40 58](https://github.com/user-attachments/assets/a5b83df5-91ec-4e82-aa9b-5bf172cd9ec2)